x-pack/filebeat/module/mysqlenterprise/audit/test/mysql_audit_test.log-expected.json

```diff
@@ -0,0 +1,1100 @@
+        "mysqlenterprise.audit.id": 0,
+        "mysqlenterprise.audit.login.user": "root",
+        "process.pid": 33038,
+        "related.user": [
+            "root"
+        ],
+        "server.user.name": "root",
+        "service.type": "mysqlenterprise",
+        "tags": [
+            "mysqlenterprise-audit"
+        ]
+    },
+    {
+        "@timestamp": "2020-10-01T19:25:51.000Z",
+        "client.domain": "localhost",
+        "event.action": "mysql-status",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "success",
+        "event.type": [
+            "access"
+        ],
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.offset": 818,
+        "mysqlenterprise.audit.class": "general",
+        "mysqlenterprise.audit.connection_id": 13,
+        "mysqlenterprise.audit.general_data.command": "Query",
+        "mysqlenterprise.audit.general_data.query": "select @@version_comment limit 1",
+        "mysqlenterprise.audit.general_data.sql_command": "select",
+        "mysqlenterprise.audit.general_data.status": 0,
+        "mysqlenterprise.audit.id": 1,
+        "mysqlenterprise.audit.login.user": "root",
+        "related.user": [
+            "root"
+        ],
+        "server.user.name": "root",
+        "service.type": "mysqlenterprise",
+        "tags": [
+            "mysqlenterprise-audit"
+        ]
+    },
+    {
+        "@timestamp": "2020-10-01T19:25:52.000Z",
+        "client.domain": "localhost",
+        "event.action": "mysql-disconnect",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "unknown",
+        "event.type": [
+            "access",
+            "end"
+        ],
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.offset": 1164,
+        "mysqlenterprise.audit.class": "connection",
+        "mysqlenterprise.audit.connection_data.connection_type": "socket",
+        "mysqlenterprise.audit.connection_id": 13,
+        "mysqlenterprise.audit.id": 0,
+        "mysqlenterprise.audit.login.user": "root",
+        "related.user": [
+            "root"
+        ],
+        "server.user.name": "root",
+        "service.type": "mysqlenterprise",
+        "tags": [
+            "mysqlenterprise-audit"
+        ]
+    },
+    {
+        "@timestamp": "2020-10-01T19:27:45.000Z",
+        "client.domain": "localhost",
+        "event.action": "mysql-connect",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "success",
+        "event.type": [
+            "access",
```

Comment:
also `connection`?

Reply:
Was a bit unsure if access, was enough, adding in connection as well.